native/libp2p_port/internal/utils/utils.go

```diff
@@ -1,11 +1,51 @@
 package utils
 
-type Config struct {
-	ListenAddr []string
-}
+import (
+	"crypto/ecdsa"
+	"errors"
+	"math/big"
+
+	"github.com/btcsuite/btcd/btcec/v2"
+	gcrypto "github.com/ethereum/go-ethereum/crypto"
+	"github.com/libp2p/go-libp2p/core/crypto"
+)
 
 func PanicIfError(err error) {
 	if err != nil {
 		panic(err)
 	}
 }
+
+// Taken from Prysm
```

Comment:
From Prysm documentation? Code? Let's add a link to the page or github blob if we don't want to go into details.